backend/contributions/tests/test_steward_permissions.py

```
@@ -380,6 +452,122 @@ def test_steward_can_update_accepted_submission_points(self):
         self.assertEqual(self.submission.converted_contribution.points, 80)
         self.assertEqual(response.data['contribution']['points'], 80)
 
+    def _create_accepted_submission(
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_create_accepted_submission`

(ANN202)